src/main/kotlin/com/careem/gradle/dependencies/main.kt

```diff
@@ -38,14 +39,19 @@ class Tldr : CliktCommand(name = "dependency-diff-tldr") {
     help = "Collapse packages with a matching group under a group.*. Collapsing " +
         "will only occur if all version numbers match. (ex --collapse com.careem.ridehail --collapse com.careem.now)."
   ).multiple()
+  private val output by option(
+    "-o",
+    "--output",
+    help = "Output type, \"plain\" and \"json\" are supported"
+  ).default("plain")
 
   private val old by argument("old.txt").path(mustExist = true)
   private val new by argument("new.txt").path(mustExist = true)
 
   override fun run() {
     val oldContents = old.readText()
     val newContents = new.readText()
-    print(tldr(oldContents, newContents, collapse))
+    print(tldr(oldContents, newContents), collapse, outputType = output.toOutputType())
```

Comment:
this print is the new print, it allows to decouple the output of tldr function and the printing logic